app/src/main/res/drawable/ic_storage_enabled.xml

```diff
@@ -1,18 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="24dp"
-    android:height="24dp"
-    android:viewportWidth="24"
-    android:viewportHeight="24"
-    android:autoMirrored="true">
-  <path
-      android:fillColor="?primaryText"
-      android:pathData="M7.2 11H6.3L6 11.3v0.9l0.3 0.3h0.9l0.3-0.3v-0.9L7.2 11z" />
-  <path
-      android:fillColor="?primaryText"
```

Comment:
Multiple of the the drawable that we are removing have a fill color based on local attribute on Fenix, are we replacing the local attribute were we are using the AC version of the icon? Because if we don't  we will be getting the `android:fillColor="@color/mozac_ui_icons_fill"`.

Reply:
Yes, see https://github.com/mozilla-mobile/fenix/pull/23313/commits/55ba4d390a58767602a547c207b292d3472a161b